app/src/main/java/com/nervousfish/nervousfish/service_locator/ServiceLocator.java

```diff
@@ -90,11 +90,43 @@
         this.qrHandler = qrHandler;
     }
 
+    IConstants initConstants() {
```

Comment:
Because they can be overriden (for example by ServiceLocatorNoNetwork). I modified the methods to make this more clear.